ipysheet/easy.py

```diff
@@ -121,7 +121,7 @@ def cell(row, column, value=0., type=None, color=None, background_color=None,
 
 def row(row, value, column_start=0, column_end=None, choice=None,
         read_only=False, format='0.[000]', renderer=None,
-        color=None, background_color=None, font_style=None, font_weight=None):
+        color=None, background_color=None, font_style=None, font_weight=None, **kwargs):
```

Comment:
No kwargs please, do the hard work and type it out :)

Reply:
Do you want to add one function parameter per attribute of `Cell` and its superclasses?
If we change just one attribute name or default value, we would need to come back to those functions and update them. Not only there are almost 15 parameters, but there are also 4 functions like that... This is a really good way to introduce bugs. And if it's just for code completion, we can live without it... That's what we do with widget constructors.

Reply:
I've put the `type` argument that was missing. But I'll keep `kwargs`, so that the user can still pass arguments to the widget constructor (e.g. layout)